### PR TITLE
feat: add pendingRequestCount to anchor summary

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Review by gvelez17 — APPROVED** [review]
great, this will be very helpful - I assume we should only call this periodically ?

File: src/repositories/request-repository.ts [hunk 1]
```diff
@@ -386,6 +386,18 @@ export class RequestRepository {
     return query
   }
 
+  /**
+   * Returns the number of pending anchor requests that remain in the database.
+   * @returns The number of requests in the database in status PENDING
+   */
+  public async countPendingRequests(): Promise<number> {
```
Comment (gvelez17) [inline src/repositories/request-repository.ts:393]: should we put a note that this is a db call and should not be made too frequently?  Or we assume people know that

Comment (stbrody) [reply src/repositories/request-repository.ts:393]: This whole RequestRepository class exists solely to contain the interactions with the request table in the database, so I think it's sufficiently implied

**Comment by stbrody** [comment]
> great, this will be very helpful - I assume we should only call this periodically ?

This will get called once at the end of every anchor batch

**Review by stephhuynh18 — APPROVED** [review]

**Pull request merged**